packages/i18n-utils/src/utils.ts

```diff
@@ -24,12 +24,12 @@ export function getPathParts( path: string ) {
 }
 
 /**
- * Checks if provided locale is a default one.
+ * Checks if provided locale is a default one - eg. en; en-GB; en-AU.
```

Comment:
Australian English (en-au) is not supported in Calypso at the moment - https://github.com/Automattic/wp-calypso/blob/trunk/packages/languages/src/languages-meta.json.